src/hokusai/hokusai.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         default: deploy
       filename:
         type: string
-        default: ./hokusai/test.yml
+        default: ""
```

Comment:
Not sure this will work as this will result in `hokusai test -f ` with no filename option given.  @zephraph do you know a nice way we can set a parameter off by default and then add the option `-f {{ parameter }}` if present?

Reply:
I added another commit, I think this is what you want?

Reply:
Yes!  I believe this will work